projects/BFDR/BirthRecord_submissionProperties.cs

```diff
@@ -5736,5 +5736,298 @@ public string InfantTransferredHelper
             }
         }
 
+
+        /// <summary>Set an emerging issue value, creating an empty EmergingIssues Observation as needed.</summary>
+        private void SetEmergingIssue(string identifier, string value)
+        {
+            if (value == null)
+            {
+                return;
+            }
+            Observation EmergingIssues = GetOrCreateObservation(identifier, CodeSystems.ObservationCode, "NCHS-required Parameter Slots for Emerging Issues", VR.IGURL.EmergingIssues, EMERGING_ISSUES_SECTION, null, identifier);
+
+            // Remove existing component (if it exists) and add an appropriate component.
+            EmergingIssues.Component.RemoveAll(cmp => cmp.Code != null && cmp.Code.Coding != null && cmp.Code.Coding.Count() > 0 && cmp.Code.Coding.First().Code == identifier);
+            Observation.ComponentComponent component = new Observation.ComponentComponent();
+            component.Code = new CodeableConcept(CodeSystems.ComponentCode, identifier, null, null);
+            component.Value = new FhirString(value);
+            EmergingIssues.Component.Add(component);
+        }
+
+        /// <summary>Get an emerging issue value.</summary>
+        private string GetEmergingIssue(string identifier)             
+        {
+            Observation EmergingIssues = GetObservation(identifier);
+            if (EmergingIssues == null)
+            {
+                return null;
+            }
+            // Remove existing component (if it exists) and add an appropriate component.
+            Observation.ComponentComponent issue = EmergingIssues.Component.FirstOrDefault(c => c.Code.Coding[0].Code == identifier);
+            if (issue != null && issue.Value != null && issue.Value as FhirString != null)
+            {
+                return issue.Value.ToString();
+            }
+            return null;
+        }
+
+
+        /// <summary>Emerging Issue Field Length 1 Number 1</summary>
+        /// <value>the emerging issue value</value>
+        /// <example>
+        /// <para>// Setter:</para>
+        /// <para>ExampleBirthRecord.EmergingIssue1_1 = "X";</para>
+        /// <para>// Getter:</para>
+        /// <para>Console.WriteLine($"Emerging Issue Value: {ExampleBirthRecord.EmergingIssue1_1}");</para>
+        /// </example>
+        [Property("Emerging Issue Field Length 1 Number 1", Property.Types.String, "emergingIssues", "One-Byte Field 1", true, VR.IGURL.EmergingIssues, false, 50)]
+        [FHIRPath("Bundle.entry.resource.where($this is Observation).where(code.coding.code='emergingissues')", "")]
```

Comment:
If the subject is child, all of these Emerging issues fields should have [FHIRSubject(FHIRSubject.Subject.Newborn)] added here. See NoCongenitalAnomaliesOfTheNewborn for an example in this file